samples/bot-sso-adaptivecard/nodejs/bots/botSSOAdaptiveCard.js

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { CardFactory, ActivityHandler } = require('botbuilder');
+const AdaptiveCardResponse = require('../resources/adaptiveCardResponseJson.json');
+const Options = require('../resources/options.json');
+
+class BotSSOAdativeCard extends ActivityHandler {
```

Comment:
you should also add code to handle adaptiveCard/action invoke and check if this token or not. If contains token, you can respond back with adaptive card saying, sso successful otherwise you can send a oauth card in response with sign in url and token exchange resource

Reply:
comment Fixed